hudi-common/src/main/java/org/apache/hudi/common/util/AvroOrcUtils.java

```diff
@@ -521,9 +522,6 @@ public static Object readFromVector(TypeDescription type, ColumnVector colVector
         } else {
           return ((BytesColumnVector) colVector).toString(vectorPos);
         }
-      case DATE:
-        // convert to daysSinceEpoch for LogicalType.Date
```

Comment:
Not sure what this patch fixed for ? Do you want to fix the NPE here ?

Reply:
Sorry, I think it's still a single PR focus feature fix, so I'll
Cancel changes here.

Reply:
done

Reply:
Still confused about what you are fixing for :(